pkg/kubelet/cm/container_manager.go

```diff
@@ -127,10 +127,11 @@ type ContainerManager interface {
 	// might need to unprepare resources.
 	PodMightNeedToUnprepareResources(UID types.UID) bool
 
-	// Implements the podresources Provider API for CPUs, Memory and Devices
+	// Implements the PodResources Provider API
 	podresources.CPUsProvider
 	podresources.DevicesProvider
 	podresources.MemoryProvider
+	podresources.DynamicResourcesProvider
```

Comment:
Just out of curiosity, why doesn't it implement the `Pods` provider? How is that different from the others?